client/src/PublicRequest.ts

```diff
@@ -26,23 +26,34 @@ export type SingleKeyResult = {
 };
 
 export type TransactionInfo = {
-    sender: Uint8Array
-    senderType: Nimiq.Account.Type
-    recipient: Uint8Array
-    value: number
-    fee: number
-    validityStartHeight: number
-    recipientType?: Nimiq.Account.Type
-    data?: Uint8Array
+    keyPath: string,
+    senderLabel?: string,
```

Comment:
I moved the `keyPath` and `senderLabel` into the `TransactionInfo` type to simplify types futher down and when altering/extending them for the swap types.